lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -135,6 +145,19 @@ def auth_ldap(self, **kwargs):
 
         self.client.auth_ldap(username, password, mount_point)
 
+    def boolean_or_cacert(self, arg):
```

Comment:
if we look at https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/etcd.py for precedence we see that it only takes in valid Python bools. I would prefer for the UX not to differ between lookup plugins for this kind of stuff.